>>> from exceptional import convert                                             
>>> convert("three four".split())                                               
Conversion succeeded! x = 34                                                    
34                                                                              
>>> convert("eleventeen".split())                                               
Conversion failed!                                                              
-1                                                                              
>>> convert(512)                                                                
Traceback (most recent call last):                                              
  File "<stdin>", line 1, in <module>                                           
  File "/Users/abingham/repos/sixty-north/pluralsight/core-python-getting-starte
d/resources/m08/visning/clip-03/use-convert-v3/exceptional.py", line 18, in conv
ert                                                                             
    for token in s:                                                             
TypeError: 'int' object is not iterable                                         
>>>                                                                             
